packages/ra-ui-materialui/src/button/SaveButton.spec.tsx

```diff
@@ -165,41 +139,6 @@ describe('<SaveButton />', () => {
         expect(onSubmit).not.toHaveBeenCalled();
     });
 
-    it('should show a notification if the form is not valid', () => {
```

Comment:
You should keep this test

Reply:
Nope, it does not show any notification anymore, we entirely rely on the form validation mechanisms

Reply:
Then it should be moved to Create & Edit tests

Reply:
As discussed, I moved the notification in the FormWithRedirect component